app/src/main/res/values-de/strings.xml

```diff
@@ -252,4 +252,5 @@
         \nSie stimmen zu, dass die Gesetze von Can Tho, Vietnam für alle Streitigkeiten gelten, die sich aus diesen Bedingungen oder den Dienstleistungen ergeben oder sich auf diese beziehen. Alle Ansprüche, die sich aus oder in Bezug auf diese Bedingungen oder die Dienstleistungen ergeben, werden ausschließlich vor den Gerichten von Can Tho City, Vietnam, geführt. Sie und SUSI erklären sich mit der persönlichen Zuständigkeit dieser Gerichte einverstanden.
         \nFür Informationen zur Kontaktaufnahme mit SUSI besuchen Sie bitte unsere Kontaktseite.</string>
     <string name="menu_item_privacy">Privatsphäre</string>
+    <string name="help_login_desc">Klicken Sie auf der Chat-Seite auf das SUSI-Logo, und Sie werden zur Skills-Seite weitergeleitet. Öffnen Sie die Menüleiste (3 vertikale Punkte) und klicken Sie auf Login. Dies führt Sie direkt zur Anmeldeseite.</string>
```

Comment:
Is this the correct translation to German?

Reply:
Yes it is correct mam

Reply:
I have just checked it